src/lib/Containers/Inbox.tsx

```diff
@@ -34,14 +36,15 @@ export class Inbox extends React.Component<Props, State> {
     }
 
     this.setState({ fetchingData: true })
-    this.props.relay.refetch(
-      {},
-      null,
-      () => {
-        this.setState({ fetchingData: false })
-      },
-      { force: true }
-    )
+
+    // Force-fetch self; this will update Active Bids but not Conversations (there will be a warning)
+    this.props.relay.refetch({}, {}, null, { force: true })
```

Comment:
Have you considered calling `refetch` on the `<ActiveBids />` component directly? You would just need to change the `createFragmentContainer` to `createRefreshContainer` in the line below. I reckon that would fix the warning as well

https://github.com/artsy/emission/blob/16677639790fdd61afa3cb14f48d19915e7741af/src/lib/Components/Inbox/ActiveBids/index.tsx#L33

Reply:
@l2succes that would get rid of the warning (it's caused by Conversations when Inbox does the force fetch). Is there a reason why the Active Bids were built as a fragment container? I don't mind changing it to a RefetchContainer but I'm curious as to whether that would have any other implications.

Reply:
I was also trying to avoid some complexity because the `_refetch` relay methods are private, and they can be called by the parent but as @alloy mentioned, it's not ideal. So I would have to make a public `refetch` method for the active bids as well. Not a ton of work, but not as simple 🤷🏻‍♀️.

Reply:
@sarahscott Right, originally the Inbox container was refreshing the data for both active bids and conversations so the `<ActiveBids />` didn't need to be a refresh container. 
The active bids component isn't being used anywhere else but in Inbox so I don't think it'll have any repercussions if we change it.

Reply:
as far as the private relay methods goes, as you said it's a bit of a dance but you can add a wrapper method on the active bids component class and call it from here.

Reply:
Curious -- what's the reason behind `_refetch` being private? Is there a public representation thats tied into some kind of data / list / container condition as opposed to being given a function to manually trigger?

Reply:
@damassi my theory is that relay is set up for child containers to update themselves as much as possible given prop or state changes - maybe it isn't ideal for parent containers to be calling child relay methods directly.